enterprise/server/raft/replica/replica.go

```diff
@@ -147,8 +153,8 @@ func (sm *Replica) Usage() (*rfpb.ReplicaUsage, error) {
 	defer iter.Close()
 
 	estimatedBytesUsed := int64(0)
-	for iter.Next() {
-		sizeBytes, err := sizeOf(iter.Value())
+	for iter.First(); iter.Valid(); iter.Next() {
```

Comment:
are there other places we should fix this?

Reply:
https://github.com/buildbuddy-io/buildbuddy/pull/2433 was the only other misuse I found